PointsInPolygon/CMakeLists.txt

```diff
@@ -23,6 +25,19 @@ set(PROJECT_SOURCES
 )
 
 
+function(multi_scopes RESULT_VARIABLE)
+
+    # here set(PARENT_SCOPE) is not usable because it will not set the
+    # variable in the caller scope but in the parent scope of the block()
+    set(${RESULT_VARIABLE} "new-value")
+    message("MY_VAR is equal: " ${RESULT_VARIABLE})
+    #return(PROPAGATE ${RESULT_VARIABLE})
+  
+endfunction()
+
+
+set(MY_VAR "initial-value")
+multi_scopes(MY_VAR)
```

Comment:
Is it PROD version?

Reply:
Remove this lines from file CMakeLisis.txt